Install.md

```diff
@@ -1,321 +1,224 @@
+# Install and utilization notes for Unix users
 
-![Sketchy iTowns usage/developing  context](Doc/Devel/Architecture/Diagrams/OslandiaiTown2Context.png)
```

Comment:
The Drawing was removed but then you still refer to it with "architecture diagram above."

Reply:
Thanks, I removed the "architecture diagram above." mention.